src/import-csv.js

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import _ from 'lodash';
+import CouchBulkImporter from 'couchbulkimporter';
+import parse from 'csv-parse';
+import BulkBadger from 'bulkbadger';
+
+export function cli (file, url, ...args) {
+  if (!file) {
+    const msg = [
+      'Usage:',
+      '',
+      'nmo import-csv [file] [couchdb-url] [...<csv options> <pairs>]',
+    ].join('\n');
+    const err = new Error(msg);
+    err.type = 'EUSAGE';
+
+    throw err;
+  }
+
+  return importcsv.apply(importcsv, [file, url, convertArgsToKey(args)]);
+}
+
+export function convertArgsToKey (args) {
+  let lastKey = '';
+
+  return args.reduce((obj, val, i) => {
+    if (i % 2 === 0) {
+      lastKey = val;
+    } else {
+      obj[lastKey] = val;
+    }
+
+    return obj;
+
+  }, {});
+
+}
+
+export function importcsv (file, url, csvOptions) {
+  return new Promise((resolve, reject) => {
+    const input = fs.createReadStream(file);
+
+    const parser = parse(_.merge({
+      delimiter: ';',
+      columns: true
+    }, csvOptions));
```

Comment:
Unfortunately cannot use the default args because if an object is passed in then it automatically overrides the default object arguments. It doesn't merge them.
I then tried Object.assign as that should work but it seems to fail with lab-babel. I have no idea why.